src/sass/_side_panel.scss

```diff
@@ -1,5 +1,21 @@
-.is-overlay {
+.l-aside.is-overlay {
+  // mobile
+  background-color: #fff;
   height: 100vh;
+  padding: 0 calc($sp-unit * 3);
+  padding-bottom: calc($sp-unit * 1.5);
+  padding-top: $sp-unit;
```

Comment:
We tend to better use the `$spv-medium`, `$spv-large`, etc. and `$sph-bar` variables. Depending on horizontal or vertical usage, pick the right one. The `$sp-unit` should stay a vanilla internal concept, especially the calculations.